spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -235,4 +235,13 @@
       expect(current_path).to eq profile_path
     end
   end
+
+  describe 'clicking the logo image during 2fa process' do
+    it 'returns them to the home page' do
+      user = build_stubbed(:user, :signed_up)
+      sign_in_user(user)
+      find("img[alt='login.gov']").click
+      expect(current_path).to eq root_path
+    end
```

Comment:
What do you think about converting this to a controller spec, so that we can easily check if the user is still signed in or not, depending on whether or not we want to sign them out? In a controller spec, you can do something like: `expect(subject.current_user).to_not be_present`. 

What I have in mind are 2 new contexts in the `describe '#new'` section at the bottom of `/spec/controllers/users/sessions_controller_spec.rb`: one where the user is not fully authenticated (`stub_sign_in_before_2fa`), and one where the user is fully authenticated (`stub_sign_in`). In the former case, we would `expect(response).to render_template(:new)`, and in the latter, `expect(response).to redirect_to profile_path`.

To make this work, we have to add `allow(controller).to receive(:user_fully_authenticated?).and_return(true)` on line 36 of `spec/support/controller_helper.rb`.